runtime/verutil/chverify.c

```diff
@@ -139,25 +134,25 @@ isInitOrClinitOrNewImpl (J9CfrConstantPoolInfo * info)
 /**
  * Determine if this name is "<init>", "<clinit>", or "<vnew>".
  *
- * @returns 0 if name is a normal name, CFR_METHOD_NAME_INIT if '<init>', CFR_METHOD_NAME_CLINIT if '<clinit>', CFR_METHOD_NAME_NEW if '<vnew>', and -1 if it starts with '<' but is not a valid class name.
+ * @returns 0 if name is a normal name, CFR_METHOD_NAME_INIT if '<init>', CFR_METHOD_NAME_CLINIT if '<clinit>', and -1 if it starts with '<' but is not a valid class name.
  * @note result is positive if the name is "<init>", "<clinit>", or "<vnew>", result is negative if the name is illegal
```

Comment:
>  `or "<vnew>", result is negative if the name is illegal`

`"<vnew>"` should be removed from this line.

Reply:
I corrected this and rebased to include the updated tests from https://github.com/eclipse-openj9/openj9/pull/19770